src/main/webapp/app/iris/exercise-chatbot/exercise-chatwidget/exercise-chat-widget.component.html

```diff
@@ -19,9 +19,27 @@ <h2>Iris</h2>
             <div *ngFor="let message of messages; let i = index">
                 <span *ngIf="i === unreadMessageIndex" class="unread-message" #unreadMessage>{{ 'artemisApp.exerciseChatbot.unreadMessages' | artemisTranslate | uppercase }}</span>
                 <div *ngFor="let content of message.content">
-                    <div [ngClass]="{ 'my-chat': message.sender === SENDER_USER, 'client-chat': message.sender === SENDER_SERVER }">
+                    <div *ngIf="message.sender === SENDER_USER" class="my-chat">
                         <span [innerHTML]="content.textContent! | htmlForMarkdown"></span>
                     </div>
+                    <div *ngIf="message.sender === SENDER_SERVER" style="width: fit-content">
+                        <div class="client-chat">
+                            <span [innerHTML]="content.textContent! | htmlForMarkdown"></span>
+                        </div>
+                        <div class="rate-message-buttons">
+                            <button style="all: unset" (click)="rateMessage(message.id, i, true)" [disabled]="message.helpful === true">
+                                <fa-icon
+                                    [class]="message.helpful === true ? 'thumbs-up-clicked' : 'clickable rate-button-not-clicked'"
+                                    style="margin-right: 15px"
+                                    [icon]="faThumbsUp"
+                                    size="sm"
+                                />
+                            </button>
+                            <button style="all: unset" (click)="rateMessage(message.id, i, false)" [disabled]="message.helpful === false">
```

Comment:
```suggestion
                            <button style="all: unset" (click)="rateMessage(message.id, i, false)" [disabled]="!message.helpful">
```

Reply:
`message.helpful` could be `undefined` also, and we only want to disable the button when it is `false`